### DISCUSSION TITLE
When using distributed tracing, why need OC_AGENT_HOST=collector.linkerd-jaeger:55678

### DISCUSSION THREAD

**Discussion by yxzhou1212** [linkerd/linkerd2#7972] [Q&A] [answered]
I saw in the version 2.11 doc
https://linkerd.io/2.11/tasks/distributed-tracing/

kubectl -n emojivoto set env --all deploy OC_AGENT_HOST=collector.linkerd-jaeger:55678

This address is already configured in linkerd proxy. Why should the application need additional configuration?
Should linkerd-proxy send the link information to Jager collector?

What is the use of this configuration？

**Comment by olix0r** [comment, marked as answer]
That configuration enables tracing in the emojivoto application so that jaeger includes spans from both the application and the proxies.

Note, also, that Linkerd will never make tracing decisions independently. That is, it's up to something else (an ingress, the application, etc) to decide to trace a given request/span. Linkerd will honor this decision and participate in tracing, but the proxy will never trace a request that does not already have trace information.

**Reply by yxzhou1212** [reply]
thanks for the response! got it .
